includes/abstracts/llms.abstract.database.store.php

```diff
@@ -20,6 +20,10 @@
  *               Hydrate before returning an array via the `to_array()` method.
  * @since 4.3.0 Add deprecated hook calls to preserve backwards compatibility for extending classes which have no `$type` property declaration.
  *              Updated the `$type` property to have a default placeholder value.
+ * @since [version] Removed deprecated items.
+ *              - `llms__created` action hook from the `LLMS_Abstract_Database_Store::create()` method
+ *              - `llms__deleted` action hook from the `LLMS_Abstract_Database_Store::delete()` method
+ *              - `llms__updated` action hook from the `LLMS_Abstract_Database_Store::update()` method
```

Comment:
```suggestion
```

We only need to go one level up when recording a removal. Since these hooks are removed from within class methods and the method records the renewal we don't need to record it on the class, too. 

If we remove a method we'd record that on the class.